### ISSUE TITLE
Dynamic properties deprecated in PHP 8.2

### ISSUE THREAD

**Issue by pd-cm** [pods-framework/pods#7134]
### Description

```
Deprecated: Creation of dynamic property Pods\REST\V1\Messages::$messages is deprecated in /wordpress/wp-content/plugins/pods/src/Pods/REST/V1/Messages.php on line 25
```

```
Creation of dynamic property PodsUI::$temp_row is deprecated in /usr/local/lsws/domain.com/html/wp-content/plugins/pods/classes/PodsUI.php on line 4865
```

`PHP v8.2, WP v6.3, Pods v2.9.19`

See https://stackoverflow.com/q/74991682
See https://wordpress.org/support/topic/deprecated-code-alert/
See https://podswp.slack.com/archives/C02SVLHQF/p1692868980832349

### Version

2.9.19

### Testing Instructions

1. Upgrade to PHP 8.2.
2. Run a rest request.
3. Plan for the future.

### Possible Workaround

The vars probably need to be explicitly declared.

**Comment by spinozarabel** [comment]
PHP ver 8.1.2 WP 6.3 Pods 2.9.19
I too get a PHP deprecated notice as follows:

PHP Deprecated:  Return type of Pods\API\Whatsit\Value_Field::offsetExists($offset) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /var/www/avasarala.in/wp-content/plugins/pods/src/Pods/API/Whatsit/Value_Field.php on line 70

**Comment by JoryHogeveen** [comment]
Hi @spinozarabel 

These errors are patched in Pods 3.0 through several patches.
Most recent: https://github.com/pods-framework/pods/pull/7146

Cheers, Jory
